### ISSUE TITLE
Extension causes high cpu load

### ISSUE THREAD

**Issue by brendanpkelleher** [zabel-xyz/plsql-language#76]
- Issue Type: `Performance`
- Extension Name: `plsql-language`
- Extension Version: `1.8.2`
- OS Version: `Darwin x64 18.5.0`
- VSCode version: `1.33.1`

:warning: Make sure to **attach** this file from your *home*-directory:
:warning:`xyz.plsql-language-unresponsive.cpuprofile.txt`

Find more details here: https://github.com/Microsoft/vscode/wiki/Explain:-extension-causes-high-cpu-load

**Comment by brendanpkelleher** [comment]
[xyz.plsql-language-unresponsive.cpuprofile.txt](https://github.com/zabel-xyz/plsql-language/files/3186131/xyz.plsql-language-unresponsive.cpuprofile.txt)

**Comment by PythooonUser** [comment]
If have also a performance issue with this extension.

VS Code Version: 1.43.2

Extension Version: 1.8.2

[xyz.plsql-language-unresponsive.cpuprofile.txt](https://github.com/zabel-xyz/plsql-language/files/4401652/xyz.plsql-language-unresponsive.cpuprofile.txt)

**Comment by mnflux** [comment]
Same problem here. Hoping this might be resolved

Version: 1.106.1 
Extension Version: 1.8.2

[xyz.plsql-language-unresponsive.cpuprofile.txt](https://github.com/user-attachments/files/23622753/xyz.plsql-language-unresponsive.cpuprofile.txt)

**Comment by PythooonUser** [comment]
I don't think this repo is maintained anymore. Development stopped 7 years ago.
